ui/app/store.js

```diff
@@ -1,16 +1,25 @@
 const createStore = require('redux').createStore
 const applyMiddleware = require('redux').applyMiddleware
 const thunkMiddleware = require('redux-thunk')
-const createLogger = require('redux-logger')
 const rootReducer = require('./reducers')
+const developmentMode = require('../../app/scripts/config').developmentMode
 
 module.exports = configureStore
 
-const loggerMiddleware = createLogger()
+const middlewares = [thunkMiddleware]
 
-const createStoreWithMiddleware = applyMiddleware(
-  thunkMiddleware,
-  loggerMiddleware
+console.log(developmentMode)
+if (developmentMode) {
+  console.log('Development Mode ON.')
+  const createLogger = require('redux-logger')
+  const loggerMiddleware = createLogger()
+  middlewares.push(loggerMiddleware)
+} else {
+  console.log('Development Mode OFF.')
+}
+
+
+const createStoreWithMiddleware = applyMiddleware(...middlewares
```

Comment:
I feel like either this should all be on one line, or `...middlewares` should get its own line.